### ISSUE TITLE
incorrect login

### ISSUE THREAD

**Issue by MrGibbage** [Medialoha/MAB-LAB#2]
I installed the new version today (thanks for the quick turnaround!). When I try to submit a report, I don't see the new report in the dashboard. In the mab-lab log, the last entry says
[27-07-13 08:35.49]     INFO    REPORT  :       New report requested !
[27-07-13 08:35.49]     DEBUG   CONFIG  :        :
[27-07-13 08:35.49]     ERROR   REPORT  :       Somebody try to access report script without a correct login/password !!!

I have tried both with and without MD5 obfuscation and have the same result. 

One interesting thing I did notice was that when changing the setting for MD5 obfuscation, the mab-lab log doesn't seem to show a setting for recording the state of the setting:

[27-07-13 08:34.56]     DEBUG   Array
(
    [a] => updateconfig
    [in-report-tags] =>
    [in-date-format] => Y-m-d H:i
    [in-date-timezone] => America/Los_Angeles
    [in-report-packagename-shrink] => 1
    [report-basicauth] => 1
    [report-basicauth-login] => XXhiddenXXX
    [report-basicauth-password] => XXXhiddenXX
    [in-report-sendmail] => 1
    [in-report-sendmail-recipients] => skip@pelorus.org
    [in-mail-from-addr] =>
    [in-mail-from-name] => MAB-LAB
    [in-db-host] => XXXhiddenXXX
    [in-db-user] => XXXhiddenXXX
    [in-db-pwd] => XXXhiddenXXX
    [in-db-name] => XXXhiddenXXX
    [in-tbl-prefix] => mabl_
)

I would have expected to see a setting in there for MD5, but I suppose you may have a way of doing this without recording the setting there, or maybe that setting just isn't being echoed to the log. Just giving you a possible clue.

The bottom line is I cannot submit a report and I do not know how to fix it.
